database/factories/EventImageFactory.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/* @var $factory \Illuminate\Database\Eloquent\Factory */
+
+use Faker\Generator as Faker;
+
+$factory->define(App\Models\EventImage::class, function (Faker $faker) {
+    return [
+        'image_path' => $faker->word,
```

Comment:
This could be replaced with a blank data image and ids should be replaced with real ids